src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -178,6 +179,29 @@ public Collection<MergeReviewConceptVersions> getMergeReviewConflictingConcepts(
 		return conflicts.values();
 	}
 
+	private boolean isTargetConceptVersionBehind(Concept sourceVersion, Concept targetVersion) {
+		Set<Integer> sourceEffectiveTimes = new HashSet<>();
+		sourceEffectiveTimes.add(sourceVersion.getReleasedEffectiveTime());
+		sourceEffectiveTimes.addAll(sourceVersion.getRelationships().stream().map(Relationship::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getReleasedEffectiveTime).collect(Collectors.toList()));
```

Comment:
Collect all release versions on the LHS and RHS. 

If the RHS does not contain all of the LHS, then it has fallen behind.

Reply:
@dmcgihtsdo You have left out language refset components.